packages/nextjs-mf/README.md

```diff
@@ -12,7 +12,7 @@ This plugin enables Module Federation on Next.js
 
 ## Supports
 
-- next ^13 || ^12(?)
+- next ^14 || ^13 || ^12(?)
```

Comment:
What does the `(?)` mean next to version 12?
Should we remove that as well?

Reply:
Ive not tested next 12 deeply but last time i did, it worked. we can probably drop the (?)

Reply:
thanks for the info. I removed the `(?)`